docs/bump.md

```diff
@@ -131,6 +132,24 @@ However, it will still update `pyproject.toml` and `src/__version__.py`.
 To fix it, you'll first `git checkout .` to reset to the status before trying to bump and update
 the version in `setup.py` to `1.21.0`
 
+
+### `--local-version`
+
+Bump the local portion of the version.
+
+```bash
+cz bump --local-version
+```
+
+For example, if we have `pyproject.toml`
+
+```toml
+[tool.commitizen]
+version = "5.3.5+0.1.0"
+```
+
+If `--local-version` is used, it will bump only the local version `0.1.0` and keep the public version `5.3.1` intact, bumping to the version `5.3.5+0.2.0`.
```

Comment:
`keep the public version `5.3.1` intact`

If I did not misunderstand, it should be `5.3.5`

Reply:
fixed!